modules/packages/LinearAlgebra.chpl

```diff
@@ -1882,7 +1984,7 @@ proc svd(A: [?Adom] ?t) throws
 proc jacobi(A: [?Adom] ?eltType, ref X: [?Xdom] eltType,
             b: [Xdom] eltType, tol = 0.0001, maxiter = 1000) {
   if Adom.rank != 2 || X.rank != 1 || b.rank != 1 then
-    halt("Wrong shape of input matrix or vector");
+    compilerError("Wrong shape of input matrix or vector");
```

Comment:
This is not part of this PR, but this error message should say that it is coming from `jacobi`

Reply:
Yeah, we are not consistent here either. I've updated the jacobi cases for this PR, but think updating all cases deserves a separate effort.